emissionsapi/openapi.yml

```
@@ -38,10 +45,18 @@ paths:
   /api/v1/average.json/:
     get:
       operationId: emissionsapi.web.get_average
-      description: Get daily average for a specified area filtered by time.
+      description: |
+          Get daily average for a specified area filtered by time.
+
+          If you provide geoframe and country and polygon, only one will be used.
```

If you provide only `geoframe and country` only one will be used as well ;-p

```suggestion
          The parameters `geoframe`, `country` and `polygon` overwrite each other so that only one will be used even if multiple parameters are provided.
```